couchbase/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # CHANGELOG - couchbase
 
-0.1.0/ Unreleased
+1.0.0/ Unreleased
 ==================
 
 ### Changes
 
-* [FEATURE] adds couchbase integration.
+* [FEATURE] adds couchbase integration to integrations-core.
+* [Improvement] Add user specified tags to service checks. See [dd-agent-3079](https://github.com/datadog/dd-agent/issues/3079). (Thanks [@arzarif][])
+
+<!--- The following link definition list is generated by PimpMyChangelog --->
```

Comment:
I do recall remi didn't really like having words like pimped in our history, so I'm not sure about his thoughts on the changelog. This was probably the case already in the dd-agent changelog, and I wouldn't like redacting the line as it is indeed the tool we're using - but we might want to check with him, it's just a comment after all.